requirements.txt

```diff
@@ -0,0 +1,2 @@
+setuptools==41.0.1
```

Comment:
It seems a little confusing to have this and the other two. I would remove this one